.github/workflows/release-main.yml

```diff
@@ -17,7 +17,12 @@ env:
 jobs:
```

Comment:
I touched this `workflow` so we can catch any errors in the release before a release. Ideally, we could refactor the common parts; for now, I think we can keep it as is. I don't envision we will change this much